cypress/integration/usePrefersColorScheme.spec.js

```diff
@@ -0,0 +1,30 @@
+/// <reference types="Cypress" />
+
+describe('usePrefersColorScheme hook', () => {
+  beforeEach(() => {
+    cy.visit('/iframe.html?path=/story/utility-usepreferscolorscheme--example-usage', {
+      onBeforeLoad(win) {
+        cy.stub(win, 'matchMedia')
+          .withArgs('(prefers-color-scheme: dark)')
+          .returns({
+            matches: true,
+          });
+      },
+    });
+    // Setting viewport dimensions to avoid side effects
+    cy.viewport(500, 500);
+  });
+
+  it('should set dark or light', () => {
+    const DARK = '(prefers-reduced-motion: dark)';
+    const LIGHT = '(prefers-reduced-motion: light)';
+
+    if (window.matchMedia(DARK).matches) {
```

Comment:
These assertions still aren't running – i think it doesn't work if they're in these `if` statements
![Screen Shot 2021-02-01 at 11 36 40 AM](https://user-images.githubusercontent.com/3903325/106488584-d9a59b80-6481-11eb-9854-500de46e145d.png)

Reply:
Reworked the test a little bit to make sure the assertions run for both light and dark